README.md

```diff
@@ -57,10 +57,6 @@ Note that the external-attacher does not scale with more replicas. Only one exte
 
 * `--leader-election`: Enables leader election. This is useful when there are multiple replicas of the same external-attacher running for one CSI driver. Only one of them may be active (=leader). A new leader will be re-elected when current leader dies or becomes unresponsive for ~15 seconds.
 
-* `--leader-election-namespace <namespace>`: Namespace where the external-attacher runs and where leader election object will be created. It is recommended that this parameter is populated from Kubernetes DownwardAPI.
```

Comment:
We still need to keep the namespace argument right?  Do we have to deprecate this as well?

Reply:
oops, yeah we need to keep this here. I'm not sure about deprecation though since you can run external-attacher outside the cluster.